Planet Descriptions updates by Improvise (#7747)
endless-sky/endless-sky: data/kahet/kahet missions.txt

event "graveyard insights"
	planet Alix

> 		add description `	 Here the Ka'het attacks were limited to the inhabited settlements; had they also destroyed the mining bases and the unmanned vessels, Parii’het would have run out of supplies thousands of years ago.`

Reviewer: This is true both here and in all other instances in the PR, but the correct apostrophe is this ', not this ’

Author: should be resolved now

	planet Celeatis
		add description `	This planet appears to have been home to one of the largest Builder research centers, which focused on studying the anomaly in the system. Had they found out how to stabilize it and use it as a wormhole, perhaps their species would have survived and spread to the rest of the Ember Waste.`
	planet Ceilia'sei

> 		add description `	Ceilia’sei’s main purpose was to act as some sort of anti-missile platform, but when the Ka’het invaded a second time, the station had no way to defend against their deadly Ravager Beams, and the station's defense systems were subsequently lost forever.`

Reviewer 2: ```suggestion
		add description `	The main purpose of Ceilia'sei was to act as some sort of anti-missile platform, but when the Ka'het invaded a second time, the station had no way to defend against their deadly Ravager Beams, and the station's defense systems were subsequently lost forever.`
```

I'd suggest switching this bit back. It's not best form, but it's not incorrect either, and it avoids making the name more complex.

> 		add description `	Ceilia’sei’s main purpose was to act as some sort of anti-missile platform, but when the Ka’het invaded a second time, the station had no way to defend against the Ka’het’s deadly Ravager Beams, and the station's defense systems were subsequently lost forever.`

Reviewer: ```suggestion
		add description `	Ceilia’sei’s main purpose was to act as some sort of anti-missile platform, but when the Ka’het invaded a second time, the station had no way to defend against their deadly Ravager Beams, and the station's defense systems were subsequently lost forever.`
```
Removing the repetition

	planet Eavine
		add description `	This moon was also home for the second Ka'het wave, which, same as the first, refused to obey their creators. It had been terraformed to make it a more appealing place to live, but after their civilization collapsed the moon began to return to its natural state. When it reverted to its original sub-zero temperatures, the facilities stopped working, and the Ka'het production in this site came to an end.`
	planet "Letis'sei"
		add description `	As it was designed to be crewed as minimally as possible, only a tiny fraction of Letis'sei station was inhabitable. The rest of the station was instead dedicated to advanced computing and targeting systems, most of which are still intact today.`
	planet Osaeli

> 		add description `	For most of the intervening millennia between the Builders’ achievement of spaceflight and their extinction, Osaeli was the de facto capital of the Builders, a key part of their economy and society. The first Het station was constructed orbiting this planet's star with the purpose of protecting them from any external threat.`

Reviewer 2: ```suggestion
		add description `	For most of the intervening millennia between the Builders' achievement of spaceflight and their extinction, Osaeli was the de facto capital of the Builders, a key part of their economy and society. The first Het station was constructed orbiting this planet's star with the purpose of protecting them from any external threat.`
```

	planet Palarei

> 		add description `	Thanks to their earlier exploration efforts, this was where the Builders encountered the first lifeforms that evolved outside their home planet. While the climate was drastically different from that of the Builders’ homeworld, the fruits the local flora produced were popular enough for Palarei to become a major farming planet.`

Reviewer 2: ```suggestion
		add description `	Thanks to their earlier exploration efforts, this was where the Builders encountered the first lifeforms that evolved outside their home planet. While the climate was drastically different from that of the Builders' homeworld, the fruits the local flora produced were popular enough for Palarei to become a major farming planet.`
```
